src/commands/ptal.ts

```diff
@@ -316,10 +316,19 @@ export default {
 		.addStringOption(option =>
 				option.setName("other")
 				.setDescription("Other links related to the PTAL, comma seperated")
-				.setRequired(false)),
+				.setRequired(false))
+		.addStringOption(option => 
+			option.setName("type")
+			.setDescription("The type of the PTAL request")
+			.setRequired(false)
+			.setChoices(
+				// space in normal is required to avoid an error for the string being empty
+				{name: "normal", value: " "},
+				{name: "baby", value: "🍼"}
+			)),
 	async execute(interaction: ChatInputCommandInteraction) {
 
-		const reply = await generateReplyFromInteraction(interaction.options.getString("description", true), interaction.options.getString("github", true), interaction.options.getString("deployment", false), interaction.options.getString("other", false), interaction);
+		const reply = await generateReplyFromInteraction(interaction.options.getString("description", true), interaction.options.getString("github", true), interaction.options.getString("deployment", false), interaction.options.getString("other", false), interaction.options.getString("type", false), interaction);
```

Comment:
```suggestion
		const reply = await generateReplyFromInteraction(interaction.options.getString("description", true), interaction.options.getString("github", true), interaction.options.getString("deployment", false), interaction.options.getString("other", false), interaction.options.getString("type", false)?.trim(), interaction);
```

Reply:
?.trim() can't be done on something that's possibly type null. Gonna add a check in the function itself